### PR TITLE
Add more benchmark tests for openxla and openxla_eval backends

### PR DESCRIPTION
**Pull request by frgossen**
Test config expansion for openxla, openxla_eval, and inductor.

### CODE REVIEW

File: test/benchmarks/test_experiment_runner.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        "--filter=^dummy$", "--dry-run"
+    ],
+                           capture_output=True,
+                           text=True)
+    expected_in_stderr = [
+        "Number of selected experiment configs: 5",
+        "Number of selected model configs: 1",
+        "'--experiment-config={\"accelerator\": \"cuda\", \"xla\": \"PJRT\", \"xla_flags\": null, \"dynamo\": \"openxla_eval\", \"test\": \"eval\"}', '--model-config={\"model_name\": \"dummy\"}'",
+        "'--experiment-config={\"accelerator\": \"cuda\", \"xla\": \"PJRT\", \"xla_flags\": null, \"dynamo\": \"openxla\", \"test\": \"train\"}', '--model-config={\"model_name\": \"dummy\"}'",
+        "'--experiment-config={\"accelerator\": \"cuda\", \"xla\": \"PJRT\", \"xla_flags\": null, \"dynamo\": \"openxla_eval\", \"test\": \"eval\"}', '--model-config={\"model_name\": \"dummy\"}'",
```
Comment (cota) [inline test/benchmarks/test_experiment_runner.py:79]:
```suggestion
        "'--experiment-config={\"accelerator\": \"cuda\", \"xla\": \"PJRT\", \"xla_flags\": null, \"dynamo\": \"openxla\", \"test\": \"eval\"}', '--model-config={\"model_name\": \"dummy\"}'",
```

File: test/benchmarks/test_experiment_runner.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        "'--experiment-config={\"accelerator\": \"cuda\", \"xla\": \"PJRT\", \"xla_flags\": null, \"dynamo\": \"openxla\", \"test\": \"eval\"}', '--model-config={\"model_name\": \"dummy\"}'",
```
Comment (cota) [inline test/benchmarks/test_experiment_runner.py:80]: This and the following should have xla:None and dynamo: Inductor, no?

Comment (cota) [reply test/benchmarks/test_experiment_runner.py:80]: I meant xla:none

File: test/benchmarks/test_experiment_runner.py
Comment (golechwierowicz) [inline test/benchmarks/test_experiment_runner.py:1]: What is this supposed to test? The set of possible configs, flags passed to subprocesses or something else?

Comment (frgossen) [reply test/benchmarks/test_experiment_runner.py:1]: Mostly that the options are expanded correctly and also that the script generally runs with a GPU.

File: test/benchmarks/test_experiment_runner.py (hunk 2 above)
Comment (cota) [reply test/benchmarks/test_experiment_runner.py:84]: err, xla:null :)

**Review by cota — APPROVED** [review]

**Review by golechwierowicz — APPROVED** [review]

**Pull request merged**